api/lib/ZwaveClient.ts

```diff
@@ -6257,25 +6252,16 @@ class ZwaveClient extends TypedEventEmitter<ZwaveClientEventCallbacks> {
 
 			if (result.reInterview) {
 				this.logNode(zwaveNode, 'info', 'Will be re-interviewed')
-				// Mark node for firmware update check after interview completes
-				// to avoid race condition with stale firmware version
-				if (!this.cfg.disableAutomaticFirmwareUpdateChecks) {
-					this._nodesPendingFirmwareUpdateCheck.add(zwaveNode.id)
-				}
 			} else {
 				// Query for new firmware updates immediately if not re-interviewing
 				// (unless automatic firmware update checks are disabled)
-				if (!this.cfg.disableAutomaticFirmwareUpdateChecks) {
-					this._checkFirmwareUpdatesAfterUpdate(zwaveNode.id).catch(
-						(error) => {
-							this.logNode(
-								zwaveNode,
-								'error',
-								`Failed to check firmware updates after update: ${error.message}`,
-							)
-						},
+				this._checkNodeFirmwareUpdates(zwaveNode.id).catch((error) => {
+					this.logNode(
+						zwaveNode,
+						'error',
+						`Failed to check firmware updates after update: ${error.message}`,
```

Comment:
here I immediately check for node fw updates when result doesn't have `reInterview` set